creepy/copy.py

```diff
@@ -0,0 +1,84 @@
+import os
```

Comment:
Is this ok for this file to be named `copy`? :)

Reply:
It's ok. The problem occurs when there are copy.py on work dir.